### PR TITLE
docs: Add minimum linux kernel version to README

### PR DESCRIPTION
**Pull request by FelixTing**
Signed-off-by: Felix Ting <felix@iotechsys.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-examples/blob/master/.github/CONTRIBUTING.md.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## New Imports
Are there any new imports or modules? If so, what are they used for and why?

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ The following devices have been tested with EdgeX:
 
 ## How does the device service work?
 - The device service ONLY works on Linux system.
+- The Linux kernel version is restricted to v5.10.x.
```
Comment (farshidtz) [inline README.md]: That is the minimum, not the only minor version.
```suggestion
- The device service ONLY works on Linux with kernel v5.10 or higher.
```

**Review by farshidtz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cloudxxx8) [reply README.md]: @farshidtz it seems someone built the binary failed in 5.11 kernel
https://github.com/edgexfoundry/device-usb-camera/issues/10

Comment (farshidtz) [reply README.md]: Maybe their libc-dev wasn't built against 5.11?

Comment (farshidtz) [reply README.md]: The APIs suppose to be backward compatible. Both Alpine 3.15 (used as the docker image base) and Ubuntu 22.04 (use as snap base) use kernel 5.15 right now, on which we can build and run the application without issues.

Comment (FelixTing) [reply README.md]: Added the suggested change. Thank you @farshidtz.

**Review by farshidtz — APPROVED** [review]
Thanks. LGTM

**Pull request merged**